src/test/java/codeu/controller/ActivityFeedServletTest.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2017 Google Inc.
```

Comment:
And finally the unit test for the servlet, which currently just makes sure the request dispatcher is being requested and forwarded with the provided instances of request and response